tests/macrodata/test_pynsee_macrodata.py

```diff
@@ -160,18 +155,21 @@ def test_get_dataset_1(self):
             self.assertTrue(isinstance(data, pd.DataFrame))
 
         def test_get_dataset_2(self):
-            data = get_dataset("CNA-2014-CPEB",
-                               filter="A.CNA_CPEB.A38-CB.VAL.D39.VALEUR_ABSOLUE.FE.EUROS_COURANTS.BRUT",
+            
+            data = get_dataset("CHOMAGE-TRIM-NATIONAL",
+                               filter="T.HALO.VALEUR_ABSOLUE.FR-D976.1.15-24.INDIVIDUS.CVS.FALSE",
                                lastNObservations=1)
             self.assertTrue(isinstance(data, pd.DataFrame))
 
-        def test_get_dataset_3(self):
-            data1 = get_dataset(
-                "IPC-2015", filter="M......ENSEMBLE...CVS.2015.")
-            data2 = get_dataset("IPC-2015", filter="M......ENSEMBLE...CVS.2015.",
-                                includeHistory=True,
-                                updatedAfter="2017-07-11T08:45:00")
-            self.assertTrue(len(data1.index) < len(data2.index))
+        # def test_get_dataset_3(self):
+        #     data1 = get_dataset(
+        #         "IPC-2015", filter="M......ENSEMBLE...CVS.2015.")
+            
+        #     data2 = get_dataset("IPC-2015", filter="M......ENSEMBLE...CVS.2015.",
+        #                         includeHistory=True,
+        #                         # updatedAfter="2017-07-11T08:45:00" disabled as API does not return anything
+        #                        )
+        #     self.assertTrue(len(data1.index) < len(data2.index))
```

Comment:
uncomment or remove?

Reply:
once I have time I will include this check back, so I prefer to leave it like this for the moment